bldsys/cmake/global_options.cmake

```diff
@@ -52,3 +52,5 @@ add_definitions(-DROOT_PATH_SIZE=${ROOT_PATH_SIZE})
 
 set(CMAKE_C_FLAGS_DEBUG   "-DDEBUG=0 ${CMAKE_C_FLAGS_DEBUG}")
 set(CMAKE_CXX_FLAGS_DEBUG "-DDEBUG=0 ${CMAKE_CXX_FLAGS_DEBUG}")
+
+add_compile_definitions($<$<CONFIG:DEBUG>:VKB_DEBUG>)
```

Comment:
We should add compile definitions per target `target_compile_definitions`

https://cmake.org/cmake/help/latest/command/target_compile_definitions.html

Reply:
Why should we do that?
VKB_DEBUG probably is a some global setting, that should not be defined per target.